### PR TITLE
Dnd5e bonus cheat sheet

### PR DESCRIPTION
**Pull request by doorknob22**
###### What does your Pull Request do (check all that apply)?

- [ x] Improvement
    - [x ] Enhancement: **`{DnD5e Proficiency}: {Fixed typos added attribute bonus}`**

###### Description of changes

Removed typo of ':" from previous commit and added information about what the attribute bonus modifier is given a character's score in that attribute.

###### People to notify (@mention interested parties)
@tagawa 

---

Instant Answer Page: https://duck.co/ia/view/dnd5e_bonus_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@doorknob22 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+	"id": "dnd5e_proficiency_cheat_sheet",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: If you agree, we can change this to `dnd5e_bonus_cheat_sheet`

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"name": "dnd5e_proficiency",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: This should be human-readable, so something like `D&D 5th Edition Bonus Cheat Sheet`

**Comment by tagawa** [comment]
Thanks for adding the new data to this cheat sheet @doorknob22 
It's looking much better. I was thinking - in future, other people might want to add more data to this so what do you think about renaming it to DnD 5e bonus cheat sheet (`dnd5e-bonus.json`), or similar? I don't know much about about D&D but it seems there's other bonus-related information that could perhaps be included later, such as stat bonus, bonus actions, etc.

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json (hunk 1 above)
Comment (doorknob22) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: Please do.

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json (hunk 2 above)
Comment (doorknob22) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: This is from when I was having trouble getting the thing to test, if it's possible to have it changed I'd love to.

**Comment by doorknob22** [comment]
@tagawa  

Not sure if this is the right way to write a reply, didn't see a reply button. Renaming to dnd 5e bonus sounds good, I'll look into doing that soon (unless I don't have enough privileges to rename it or something).

If there's other information I think of that could easily fit I'll add it here. If not I'll make a new cheatsheet or whatever else is appropriate as ideas come to me. Thanks for your help.

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json (hunk 1 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: Please could you change this in your file? After you've saved it, run the `git commit` and `git push` commands again and this pull request will be updated automatically. I can then update the ID in the Instant Answer page.

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json (hunk 2 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: The same as above, you should be able to make the change and the pull request will be updated automatically. Please let me know if you need any help with the Git commands.

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
+{
+	"id": "D&D 5th Edition Bonus Cheat Sheet",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: Thanks for changing the name but this ID should be a computer-readable version, in other words with underscores and no spaces. Please could you try changing the ID to `dnd5e_bonus_cheat_sheet` and then I'll update the Instant Answer page so it should work.

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json [hunk 4]
```diff
@@ -0,0 +1,96 @@
+{
+	"id": "D&D-5th-Edition-Bonus-Cheat-Sheet",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: Thanks for doing this @doorknob22 but I'm afraid they have to be underscores, not hyphens. Please could you change it to `dnd5e_bonus_cheat_sheet` (lowercase as well). You can just copy and paste this:

    "id": "dnd5e_bonus_cheat_sheet",

**Comment by tagawa** [comment]
Hi again @doorknob22. Please could you also change the filename to `dnd5e-bonus.json` (with a hyphen, not underscore). Sorry for not spotting this earlier. Hopefully this will then make it work.

**Comment by tagawa** [comment]
Hi @doorknob22. Sorry for not being clear - please could you change the filename to `dnd5e-bonus.json` ?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=dnd5e bonus cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/dnd5e_bonus_cheat_sheet_mobile.png?7iAJvW1Xuu)

**Comment by tagawa** [comment]
Yay, we got there! The cheat sheet is now on the beta server for testing: https://beta.duckduckgo.com/?q=dnd5e+bonus+cheat+sheet&ia=cheatsheet&iax=1

Thank you for being patient with all the changes @doorknob22. Let's keep it open for a few days to see if there are any other review comments.

**Comment by tagawa** [comment]
@doorknob22 Well no further review comments and it looks good on the beta server, so let's merge! Thanks again for making this cheat sheet and for all the edits to make it work. It should go live in a few days time. 🎈

**Comment by daxtheduck** [comment]
@doorknob22 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**